site/README.md

```diff
@@ -27,6 +27,17 @@ Access the site at [http://localhost:1313](http://localhost:1313). Press `Ctrl-C
 
 The [site/content/docs/latest](./content/docs/latest) directory holds the project documentation whereas [site/themes/template/static../img/docs](./themes/template/static../img/docs) contains the images used in the documentation. Note they have to be under that folder to be properly served.
 
+#### Run Hugo with Docker
+
+To ease the local development and prevent you from polluting your local environment with tools that rarely use, 
+we have added the option of running the `Hugo` server via `Docker` through a `Make` target.
```

Comment:
For development there is a new style guide that contributors must start following ([see here](https://github.com/vmware-tanzu/kubeapps/blob/main/site/content/docs/latest/reference/developer/style-guide.md#avoid-using-we)). It recommends avoiding "we".